packages/dialtone-icons/README.md

```diff
@@ -20,77 +20,35 @@ Once installed, you can import icons in a couple different ways:
 // named import (preferred on vite)
 import { DtIconAccessibility } from '@dialpad/dialtone-icons/vue2'; // Vue 2.6+
 import { DtIconAccessibility } from '@dialpad/dialtone-icons/vue3'; // Vue 3+
+import { DtIllustrationBlankSpace } from '@dialpad/dialtone-icons/vue2'; // Vue 2.6+
+import { DtIllustrationBlankSpace } from '@dialpad/dialtone-icons/vue3'; // Vue 3+
 
 // default import (preferred on webpack)
 import DtIconAccessibility from '@dialpad/dialtone-icons/vue2/accessibility'; // Vue 2.6+
 import DtIconAccessibility from '@dialpad/dialtone-icons/vue3/accessibility'; // Vue 3+
+import DtIllustrationBlankSpace from '@dialpad/dialtone-icons/vue2/blank-space'; // Vue 2.6+
+import DtIllustrationBlankSpace from '@dialpad/dialtone-icons/vue3/blank-space'; // Vue 3+
 ```
 
 - All the icon components:
 
 ```js
-import * as icons from '@dialpad/dialtone-icons/vue2'; // Vue 2.6+
-import * as icons from '@dialpad/dialtone-icons/vue3'; // Vue 3+
+import { icons } from '@dialpad/dialtone-icons/vue2'; // Vue 2.6+
+import { icons } from '@dialpad/dialtone-icons/vue3'; // Vue 3+
```

Comment:
I'm a bit confused what this is, is this an array of icon vue components?

Reply:
Yes, that's mostly for compatibility reasons, we're already importing those on DtIcon and maybe on other places, it includes all the icons.

Reply:
alright no prob just wasn't sure of it's purpose